app/helpers/form_inputs_helper.rb

```diff
@@ -11,17 +11,31 @@ def employee_address
 end
 
 def education
-	[
-		['No High School Diploma', 'No High School Diploma'],
-		['High School Diploma', 'High School Diploma'],
-		['GED', 'GED'],
-		['Vocational Certificate', 'Vocational Certificate'],
-		['Some College/Never Completed', 'Some College/Never Completed'],
-		['Associates Degree', 'Associates Degree'],
-		["Bachelor's Degree", "Bachelor's Degree"],
-		["Master's Degree", "Master's Degree"],
-		['Doctoral Degree', 'Doctoral Degree']
-	]
+	if I18n.locale != I18n.default_locale
+		[
+			['Sin Diploma de Escuela Secundaria', 'No High School Diploma'],
```

Comment:
`['Sin Diploma de Escuela Secundaria', 'Sin Diploma de Escuela Secundaria']` would make the spanish version save to the db, then it would persist o the show page.  But then we're stuck with it that way?